steely/plugins/define.py

```diff
@@ -42,18 +44,39 @@ def main(bot, author_id, message, thread_id, thread_type, **kwargs):
             bot.sendMessage(ANGRY_STRING, thread_id=thread_id,
                             thread_type=thread_type)
             return
+    elif command == 'image':
+        if ' ' in text:
+            command, image_url = text.split(' ', 1)
+            try:
+                bot.sendRemoteImage(
```

Comment:
the nested logic here is doirty, could be worth moving everything to functions so it's clearer

Reply:
can do friend